src/jemalloc.c

```diff
@@ -1855,7 +1860,8 @@ malloc_init_hard_a0_locked(void) {
 	if (config_prof) {
 		prof_boot0();
 	}
-	malloc_conf_init(&sc_data, bin_shard_sizes);
+	char readlink_buf[PATH_MAX + 1];
```

Comment:
Explicitly set `readlink_buf[0]` = `\0` instead of relying on the downstream to do so, given that we need `strlen` on it later.